### PR TITLE
More multilanguage screens

### PR DESCRIPTION
**Pull request by Rand3rino**
I added German to the screen titles and the inspection info. Can anyone verify the words I used? locales/DE_DE.json has the definitions.

### CODE REVIEW

File: locales/DE_DE.json [hunk 1]
```diff
@@ -1,26 +1,36 @@
 {
   "boxInfo": {
     "boxNumber": "Nistkasten Nummer",
-    "date": "Datum",
+    "date": "Inspektionsdatum",
     "boxCondition": "Zustand Kasten",
     "boxNumberNew": "Nistkasten Nummer Neu",
     "cleaned": "Gereinigt",
     "occupied": "Desetzt",
```
Comment (SimonOyen) [inline locales/DE_DE.json]: "Besetzt" instead of "Desetzt" here

File: locales/DE_DE.json [hunk 2]
```diff
@@ -1,26 +1,36 @@
...
     "eggCount": "Anzahl Eier",
-    "chickCount": "Anzalh Jungvögel",
+    "chickCount": "Anzahl Jungvögel",
     "chickAge": "Alter Jungvögel",
     "specie": "Vogelart",
-    "ringed": "Berignt",
-    "comments": "Bemerkungen"
+    "ringed": "Anzahl Beringter Bungvögel",
```
Comment (SimonOyen) [inline locales/DE_DE.json]: should be "Anzahl beringter Jungvögel"

File: locales/DE_DE.json (hunk 1 above)
Comment (Rand3rino) [reply locales/DE_DE.json]: Fixed

File: locales/DE_DE.json [hunk 3]
```diff
@@ -1,26 +1,36 @@
...
     "password": "Passwort",
     "logInButton": "Einloggen"
   },
-  "main": {
-    "sample": "",
-    "save": ""
+  "screenName": {
+    "map": "Landkarte",
+    "nestingBox": "Nistkasten",
+    "boxInfo": "Kast Information",
```
Comment (SimonOyen) [inline locales/DE_DE.json]: should be "Informationen zum Nistkasten"

File: locales/DE_DE.json [hunk 4]
```diff
@@ -1,26 +1,36 @@
...
+    "inspectionInfo": "Kontrolle Information",
```
Comment (SimonOyen) [inline locales/DE_DE.json]: should be "Informationen zur Inspektion"

File: locales/DE_DE.json (hunk 2 above)
Comment (Rand3rino) [reply locales/DE_DE.json]: Fixed

File: locales/DE_DE.json [hunk 5]
```diff
@@ -1,26 +1,36 @@
...
+    "inspectionNew": "Kontrolle Neu",
```
Comment (SimonOyen) [inline locales/DE_DE.json]: should be "Neue Inspektion"

File: locales/DE_DE.json [hunk 6]
```diff
@@ -1,26 +1,36 @@
...
+    "boxNew": "Kasten Neu",
```
Comment (SimonOyen) [inline locales/DE_DE.json]: should be "Neuer Niskasten"

File: locales/DE_DE.json (hunk 3 above)
Comment (Rand3rino) [reply locales/DE_DE.json]: Fixed

**Review by SimonOyen — APPROVED** [review]
Well done!

**Comment by Rand3rino** [comment]
Thank you! I learned some grammar rules today

**Pull request merged**